stl/inc/ratio

```diff
@@ -43,35 +45,36 @@ struct _Safe_addX<_Ax, _Bx, false, false> { // _Ax + _Bx would overflow
 };
 
 template <intmax_t _Ax, intmax_t _Bx>
-struct _Safe_add : _Safe_addX<_Ax, _Bx, _Sign_of<_Ax>::value != _Sign_of<_Bx>::value,
-                       (_Abs<_Ax>::value <= INTMAX_MAX - _Abs<_Bx>::value)>::type {
+struct _Safe_add : _Safe_addX<_Ax, _Bx, _Sign_of(_Ax) != _Sign_of(_Bx), _Abs(_Ax) <= INTMAX_MAX - _Abs(_Bx)>::type {
     // computes _Ax + _Bx, forbids overflow
 };

```

Comment:
Both `_Safe_addX` and `_Safe_add` are simple class templates with no specializations that exist only to map to their base types - I suspect both of these can be replaced with more efficient alias templates.

Reply:
I suspect that I could just replace it with a `constexpr` function, since those aren't used for SFINAE.